physionet-django/user/models.py

```diff
@@ -778,6 +778,21 @@ class CredentialApplication(models.Model):
     class Meta:
         default_permissions = ('change',)
 
+    def get_training_status(self):
+        training_status = self.user.trainings.all().filter(
+            user__username__icontains=self.user.username).values_list('status', flat=True).first()
+        return self.get_training_status_name(training_status)
+
+    def get_training_status_name(self, training_status):
```

Comment:
You could use the Enum class to get the text representation of the status, e.g.:

```
status = 1
print(TrainingStatus(status).name)
>> WITHDRAWN
```

Reply:
That's a good point, @tompollard. When I wrote the code, I saw that - for the Review tab on the training list -, there was only the "CITI Data or Specimens Only Research" training type for all users. So I added the columns with only the "CITI Data or Specimens Only Research" in mind. 

But you are right, we need to figure out whether it makes sense to keep this information because it's still not clear to me what we should show on the Training Status column when we have different types of training.

Reply:
Thanks, @tompollard.